test/Microsoft.TemplateEngine.IDE.IntegrationTests/SnapshotTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NET
+
+using Microsoft.Extensions.Logging;
+using Microsoft.TemplateEngine.Authoring.TemplateApiVerifier;
+using Microsoft.TemplateEngine.Authoring.TemplateVerifier;
+using Microsoft.TemplateEngine.TestHelper;
+using Microsoft.TemplateEngine.Tests;
+using Xunit.Abstractions;
+
+namespace Microsoft.TemplateEngine.IDE.IntegrationTests
+{
+    public class SnapshotTests : TestBase
```

Comment:
@YuliiaKovalova - this https://github.com/dotnet/templating/pull/5908 needs to be backportet as well, otherwise random CI issues can occur